source/NVDAObjects/UIA/__init__.py

```diff
@@ -297,6 +297,20 @@ def __init__(self,obj,position,_rangeObj=None):
 		elif position==textInfos.POSITION_LAST:
 			self._rangeObj=self.obj.UIATextPattern.documentRange
 			self.collapse(True)
+		elif position==textInfos.POSITION_FIRSTVISIBLE:

```

Comment:
I think I'd prefer this being more compact.

```suggestion
		elif position in (textInfos.POSITION_FIRSTVISIBLE, textInfos.POSITION_LASTVISIBLE):
```